include/antidote.hrl

```diff
@@ -182,10 +175,11 @@
 -type op() :: {op_name(), op_param()}.
 -type effect() :: term().
 
+-type microsecond() :: non_neg_integer().
 
 %% DC Id is the riak_core ring cluster name
--type dcid() :: undefined | riak_core_ring:riak_core_ring().
--type snapshot_time() :: 'undefined' | vectorclock:vectorclock().
+-type dcid() :: undefined | {term(), term()}.
+-type snapshot_time() :: vectorclock:vectorclock().
```

Comment:
I checked all usages of snapshot_time() 
-> undefined is not allowed except in some places of logging_vnode.erl
-> added undefined at places where it is explicitly allowed (see logging_vnode.erl changes)